fastlane/iOS.Fastfile

```diff
@@ -1,29 +1,14 @@
 platform :ios do
   lane :check_version_code_exists do
     testflight_latest = latest_testflight_build_number(
-      app_identifier: ENV["APP_ID_IOS"]
+      app_identifier: ENV["APP_ID"]
     )
 
     versions = Array(testflight_latest)
 
     UI.user_error!("Version code #{versions} has already been used!") if Array(versions).include? ENV['APP_VERSION_CODE'].to_i
   end
 
-  # Saving this for later, but may not need anymore
-  lane :set_version do
-    # Set the version name from the environment
-    increment_version_number(
```

Comment:
After reviewing with @dsamojlenko looks like we need to put this back - but we'll test further.

Reply:
Couldn't find another fix, so brought this code back